src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/RcaConf.java

```diff
@@ -197,21 +204,29 @@ public DeciderConfig getDeciderConfig() {
     return ImmutableMap.copyOf(conf.getRcaConfigSettings());
   }
 
+  public <T> T readRcaConfig(String rcaName, String key, T defaultValue, Class<? extends T> clazz) {
```

Comment:
Can we reuse the Config<T> class to read config settings here ?

Reply:
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/issues/411